src/bql.py

```diff
@@ -322,6 +322,15 @@ def execute_phrase(bdb, phrase, bindings=()):
                             'Invalid stattype: %s' % (cmd.stattype))
                     else:
                         stattype = cmd.stattype
+                    # Check that strings are not being modeled as numerical.
+                    if stattype == 'numerical':
+                        cursor = bdb.sql_execute(
+                                'SELECT %s FROM %s' % (qc, qt))
```

Comment:
I think you can also write `SELECT COUNT(*) FROM qt WHERE typeof(qc) == 'text'`. @riastradh-probcomp which approach is better for checking for strings in the table?